### PR TITLE
Add configs of engine event logger for each engine

### PR DESCRIPTION
**Pull request by zhaomin1423**
### _Why are the changes needed?_

The default value of kyuubi.engine.event.loggers is SPARK, but it can't be used outside the spark engine, so we add the config for each engines.
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by zhaomin1423** [comment]
want to merge it to 1.6.0. @pan3793

**Comment by zhaomin1423** [comment]
The setting.md will be update later.

File: docs/deployment/settings.md [hunk 1]
```diff
@@ -346,6 +347,13 @@ kyuubi.ha.zookeeper.quorum||(deprecated) The connection string for the zookeeper
 kyuubi.ha.zookeeper.session.timeout|60000|The timeout(ms) of a connected session to be idled|int|1.0.0
 
 
+### Hive
```
Comment (yaooqinn) [inline docs/deployment/settings.md:350]: why is hive special?

**Comment by ulysses-you** [comment]
@zhaomin1423  can you resolve the failed test ?  thank you

**Review by ulysses-you — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

**Pull request closed without merging**